test/src/pages/ios/welcome.page.js

```diff
@@ -1,10 +1,21 @@
 const AndroidWelcomePage = require('../android/welcome.page');
 
-class WelcomePage extends AndroidWelcomePage{
-        masterKeyTitle =
-            '~Your Master Key';
+class WelcomePage extends AndroidWelcomePage {
+    masterKeyTitle =
+        '~Your Master Key';
         mnemonic =
-            '//XCUIElementTypeStaticText[contains(@label,"word")]';
+        '//XCUIElementTypeStaticText[contains(@label,"word")]';
+    wordSuggestion = '~selectSuggestion0';
+    typeNemonic(position) {
+        return `(//XCUIElementTypeTextField[@name="input.wordInput"])[${position}]`;
+    }
+    pin(num) {
+        return `//XCUIElementTypeOther[@name="keypad_${num}"]`;
+    }
+
+
+
+    landing ='(//XCUIElementTypeOther[@name="0x4cf3...Fa0e"])[1]';
```

Comment:
if it's a temporal locator as it looks like, please add a TODO statement (`// TODO:`) just before it